libraries/aws_dynamodb_table.rb

```diff
@@ -64,6 +66,11 @@ def initialize(opts = {})
     end
   end
 
+  def encrypted?
+    return false unless @dynamodb_table[:sse_description]
+    @dynamodb_table[:sse_description][:status].upcase == 'ENABLED'
```

Comment:
This could probably be 
```
@dynamodb_table.dig(:sse_description, :status).upcase == 'ENABLED' || false
```

Reply:
Looks good however a user could leave out the sse_description completely as its optional, the dig would return nil and calling upcase would get an undefined method